### PR TITLE
Add a link to a Python implementation of the standard

### PR DESCRIPTION
**Pull request by mecforlove**
Hello, thanks for your awesome encoding idea. I went through your code and wrote the Python version.

### CODE REVIEW

**Comment by mecforlove** [comment]
@keith-turner Please deal with this pr. Thanks.

**Comment by keith-turner** [comment]
@mecforlove sorry for the delay.   I want to look at your project before adding the link and have not had the time.  I will do this shortly.

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ Libraries [implementing](docs/encoding.md) the Ecoji encoding standard. Submit P
 | [Java](https://github.com/netvl/ecoji-java) | Implementation of Ecoji written in Java, usable in any JVM language.
 | [PHP](https://github.com/Rayne/ecoji-php) | PHP 7.x implementation of Ecoji. Available as [`rayne/ecoji` on Packagist](https://packagist.org/packages/rayne/ecoji).
 | [Rust](https://github.com/netvl/ecoji.rs) | Implementation of Ecoji written in the Rust programming language.
+| [Python](https://github.com/mecforlove/ecoji) | Implementation of Ecoji written in the Python3 programming language.
```
Comment (Rayne) [inline README.md]: That's a wrong link. It should be https://github.com/mecforlove/ecoji-py instead.

**Review by Rayne — CHANGES_REQUESTED** [review]

**Comment by mecforlove** [comment]
@Rayne sorry, I fixed it.

**Review by Rayne — APPROVED** [review]

**Comment by keith-turner** [comment]
@mecforlove I spent a while testing your implementation and it works great!   I need to write an automated test suite so I Can more quickly test different implementation.  Again, sorry for the delay.  Your implementation is very nice!

**Pull request merged**